WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/PostsCardViewModel.swift

```diff
@@ -212,7 +214,7 @@ private extension PostsCardViewModel {
             self?.syncing = nil
             self?.performInitialLoading()
             self?.refresh()
-        }, failure: { [weak self] _ in
+        }, failure: { [weak self] error in
```

Comment:
[nit] We aren't using `error` so we can revert back to `_`

```suggestion
        }, failure: { [weak self] _ in
```